src/components/CreateVirtualCardsFromEmails.js

```diff
@@ -0,0 +1,258 @@
+        <Link route="editCollective" params={{ slug: this.props.collectiveSlug, section: 'payment-methods' }}>
+          <StyledButton buttonSize="large" mt="2em" justifyContent="center">
+            <FormattedMessage
+              id="virtualCards.create.requirePM"
+              defaultMessage="You must add a payment method to your account to create gift cards"
+            />
+          </StyledButton>
+        </Link>
+      </Flex>
+    );
+  }
+
+  renderSuccess(createdVirtualCards) {
+    return (
+      <Flex flexDirection="column" alignItems="center" justifyContent="center">
+        <P color="green.700">
+          <CheckCircle size="3em" />
+        </P>
+        <FormattedMessage
+          id="virtualCards.create.successSent"
+          defaultMessage="Your {count} gift cards have been sent! Go back to the {link} to see them."
+          values={{
+            count: createdVirtualCards.length,
+            link: (
+              <Link route="editCollective" params={{ slug: this.props.collectiveSlug, section: 'gift-cards' }}>
+                <FormattedMessage id="virtualcards.create.listPage" defaultMessage="gift cards list" />
+              </Link>
+            ),
+          }}
+        />
+      </Flex>
+    );
+  }
+
+  render() {
+    const loading = get(this.props, 'data.loading');
+    const paymentMethods = get(this.props, 'data.Collective.paymentMethods', []);
+
+    if (loading) return <Loading />;
+    if (paymentMethods.length === 0) return this.renderNoPaymentMethodMessage();
+
+    const { submitting, values, errors, createdVirtualCards, serverError } = this.state;
+
+    return createdVirtualCards ? (
+      this.renderSuccess(createdVirtualCards)
+    ) : (
+      <form ref={this.form} onSubmit={this.onSubmit}>
+        <Flex flexDirection="column">
+          <InlineField
+            name="amount"
+            label={<FormattedMessage id="virtualCards.create.amount" defaultMessage="Amount" />}
+          >
+            <StyledInputAmount
+              id="virtualcard-amount"
+              currency={this.props.currency}
+              onChange={e => this.onChange('amount', e.target.value)}
+              error={this.getError('amount')}
+              value={values.amount}
+              min={MIN_AMOUNT}
+              disabled={submitting}
+              required
+            />
+          </InlineField>
+
+          <InlineField
+            name="paymentMethod"
+            label={<FormattedMessage id="virtualCards.create.paymentMethod" defaultMessage="Payment Method" />}
+          >
+            <StyledPaymentMethodChooser
+              disabled={submitting}
+              paymentMethods={paymentMethods}
+              defaultPaymentMethod={this.getDefaultPaymentMethod()}
+              onChange={option => this.onChange('paymentMethod', option)}
+            />
+          </InlineField>
+
+          <Flex flexDirection="column" mb="3em">
+            <label style={{ width: '100%' }}>
```

Comment:
In the future, you can use something like `<Box as="label" width={1}></Box>` to avoid the inline style declaration and still render a `label` element.